src/utils/promiseTimeout.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * promiseTimeout - Reject a promise if it doesn't resolve after a given timeout window.
+ *
+ * @param {number} ms Timeout (ms)
+ * @param {Promise} promise Promise to timeout
```

Comment:
Missing doc for `message` param.

Reply:
Also, I'd suggest making the `message` param come last and optional and provide a default message in the case it's not provided.

Reply:
updated